README.md

```diff
@@ -76,6 +76,7 @@
 - [`@kevinwolf/formal`](https://github.com/kevinwolfcr/formal) Elegant form management primitives for the react hooks era.
 - [`@koale/useworker`](https://github.com/alewin/useWorker) ⚙️ Running heavy task in background using web workers, without blocking the UI
 - [`@marvelsq/use-properties-hook`](https://github.com/MarvelSQ/use-properties-hook) Instance functions inside FunctionComponent like `class-properties` and equal in `ShallowCompare`
+- [`@react-hookz/web`](https://github.com/react-hookz/web) - A library of general-purpose React hooks built with care and SSR compatibility in mind.
```

Comment:
Can you move it to the catalogs section?

Reply:
Done! Also moved an item that I noticed was out of alphabetical order.